src/samples/UWP/Uno.Themes.Samples.Shared/Content/NestedSamples/MediaPlayerElementSample_NestedPage5.xaml

```diff
@@ -0,0 +1,95 @@
+							  Style="{StaticResource MaterialAppBarButton}">
+					<AppBarButton.Icon>
+						<BitmapIcon UriSource="ms-appx:///Assets/Close.png"
+									ShowAsMonochrome="False" />
+					</AppBarButton.Icon>
+				</AppBarButton>
+			</toolkit:CommandBarExtensions.NavigationCommand>
+		</CommandBar>
+
+		<TextBlock Grid.Row="1"
+				   Text="MediaPlayerElement sample using a Playlist"
+				   Style="{StaticResource MaterialTitleLarge}"
+				   Foreground="{ThemeResource OnBackgroundBrush}"
+				   Opacity="0.8"
+				   Margin="16,32,0,0" />
+
+		<ScrollViewer Grid.Row="2"
+					  Padding="16"
+					  HorizontalAlignment="Center">
+			<smtx:XamlDisplay UniqueKey="MediaPlayerElementSample_NestedPage5_UsingPlaylist"
+							  Style="{StaticResource ExpandingXamlDisplayStyle}">
+				<StackPanel>
+					<TextBlock Text="MediaPlayerElement using a playlist of three items:" />
+
+					<TextBlock Text="- Mobile Development in Visual Studio Code with Uno Platform or DotNetMAUI (.mp4)" />
+
+					<TextBlock Text="- Getting Started with Uno Platform and Visual Studio Code (.mp3 [Audio only])" />
+
+					<TextBlock Text="- Getting Started with Uno Platform and Visual Studio Code (.mp4)"
+							   Margin="0,0,0,10" />
+
+					<Border>
+						<MediaPlayerElement x:Name="MediaPlayerElementSample5"
+											AreTransportControlsEnabled="True"
+											AutoPlay="True"
+											MaxWidth="720">
+							<MediaPlayerElement.TransportControls>
+								<MediaTransportControls IsNextTrackButtonVisible="True"
+														IsPreviousTrackButtonVisible="True"
+														IsRepeatButtonVisible="True"
+														IsRepeatEnabled="True"
+														IsSkipBackwardButtonVisible="True"
+														IsSkipBackwardEnabled="True"
+														IsSkipForwardButtonVisible="True"
+														IsSkipForwardEnabled="True"
+														IsZoomButtonVisible="False"
+														IsZoomEnabled="False"
+														IsStopButtonVisible="False"
+														IsStopEnabled="False" />
+							</MediaPlayerElement.TransportControls>
+						</MediaPlayerElement>
+
+						<!--
+							With the following C# code:
+							
+							private void InitializePlaybackList()
+							{
+							var mediaPlaybackList = new MediaPlaybackList();
+							
+							mediaPlaybackList.Items.Add(new MediaPlaybackItem(MediaSource.CreateFromUri(new Uri("https://uno-assets.platform.uno/tests/videos/Mobile_Development_in_VS_Code_with_Uno_Platform_orDotNetMAUI.mp4"))));
+							mediaPlaybackList.Items.Add(new MediaPlaybackItem(MediaSource.CreateFromUri(new Uri("https://uno-assets.platform.uno/tests/audio/Getting_Started_with_Uno_Platform_and_Visual_Studio_Code.mp3"))));
+							mediaPlaybackList.Items.Add(new MediaPlaybackItem(MediaSource.CreateFromUri(new Uri("https://uno-assets.platform.uno/tests/videos/Getting_Started_with_Uno_Platform_and_Visual_Studio_Code.mp4"))));
+							
+							MediaPlayerElementSample5.MediaPlayer.Source = mediaPlaybackList;
+							}
+						-->
```

Comment:
is this needed?

Reply:
yeah, you can remove this @nguyeenvy as we are adding it only for Uno.Gallery to showcase the code-behind in XamlDisplay as it is a showcase app. But Uno.Themes sample for tests does not need it.